src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
The function should include validation to handle cases where `sourceString` is not a valid string or is empty. Consider adding a check at the beginning of the function to return an empty object if `sourceString` is falsy.